SOOUM/SOOUM/Presentations/Onboarding/NicknameSetting/Views/OnboardingNicknameTextFieldView.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  OnboardingNicknameTextFieldView.swift
+//  SOOUM
+//
+//  Created by JDeoks on 11/7/24.
+//
+
+import UIKit
+
+import ReactorKit
+import RxCocoa
+import RxGesture
+import RxSwift
+
+import SnapKit
+import Then
+
+class OnboardingNicknameTextFieldView: UIView {
+    /// 닉네임 입력 텍스트 필드
+    let textField = UITextField().then {
+        $0.typography = .init(
+            fontContainer: BuiltInFont(
+                size: 15,
+                weight: .regular
+            ),
+            lineHeight: 24,
+            letterSpacing: 0
+        )
+        $0.placeholder = "닉네임을 입력해주세요"
+        $0.textColor = .som.black
+    }
+    
+    /// 클리어버튼 터치 영역 잡는 뷰
+    let clearButtonView = UIView().then {
+        $0.backgroundColor = .clear
+    }
+    
+    let clearImageView = UIImageView().then {
+        $0.image = .cancelOutlined
+        $0.tintColor = .som.black
+    }
+    
+    var disposeBag = DisposeBag()
+    
+    // MARK: - init
+    convenience init() {
+        self.init(frame: .zero)
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.layer.cornerRadius = 12
+        self.backgroundColor = .som.gray50
+        initConstraint()
+        
+        self.rx.tapGesture()
+            .when(.recognized)
+            .subscribe(onNext: { gesture in
+                self.textField.becomeFirstResponder()
+            })
+            .disposed(by: self.disposeBag)
```

Comment:
```suggestion
        
        self.rx.tapGesture()
            .when(.recognized)
            .subscribe(with: self) { object, _ in
                object.textField.becomeFirstResponder
            }
            .disposed(by: self.disposeBag)
```
메모리 누수를 방지하기 위한 subscribe(with: self) 사용과 사용하지 않는 변수를 _ 로 명시하는 것이 좋습니다.